yarn.lock

```diff
@@ -4277,16 +4277,7 @@ stop-iteration-iterator@^1.0.0:
   dependencies:
```

Comment:
Why the yarn.lock is changed ?

Reply:
It looks to me an automatic optimization.
It merged the field that were equals.

Reply:
I guess bc i added a dependency to test the sha calculation and then i remove it so i rebuilt the yarn.lock

Reply:
Reverted